pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1952,7 +1957,9 @@ func DomainCreate(status types.DomainStatus) (int, string, error) {
 	if status.IsContainer {
 		// Use rkt tool
 		log.Infof("Using rkt tool ... ContainerImageID - %s\n", status.ContainerImageID)
-		domainID, podUUID, err = rktRun(status.DomainName, status.ContainerImageID, filename)
+		aciFileName := filepath.Join(types.VerifiedAppImgDirname, status.ImageID.String(), status.ContainerImageID)
```

Comment:
Same here.. Store the aciFileName in the Status.. Will be useful for debugging as well..